core/space/services/services_fs.go

```diff
@@ -548,6 +548,15 @@ func (s *Space) addFile(ctx context.Context, sourcePath string, targetPath strin
 			return domain.AddItemResult{}, err
 		}
 
+		//add current user as admin on the remote file
+		err = s.tc.AddCurrenUserAsFileOwner(ctx, b, targetPathBucket)
```

Comment:
Maybe this can be an internal call of the Textile Client that gets called inside `UploadFileToHub` to avoid adding more methods to the public interface of the Client.

Reply:
Good call.  Fixed in [a8ad2ff](https://github.com/FleekHQ/space-daemon/pull/182/commits/a8ad2fffb5d31c798ecc794ffa82718ab9852542).